src/qt/window.cpp

```diff
@@ -937,12 +937,15 @@ void wxWindowQt::SetAcceleratorTable( const wxAcceleratorTable& accel )
     wxWindowBase::SetAcceleratorTable( accel );
 
     // Disable previously set accelerators
-    while ( !m_qtShortcuts->isEmpty() )
-        delete m_qtShortcuts->takeFirst();
+    if(m_qtShortcuts)
```

Comment:
This was already applied, please rebase on latest master to avoid merge conflicts.